localstack/plugins.py

```diff
@@ -2,20 +2,10 @@
 
 from localstack import config
 from localstack.runtime import hooks
-from localstack.utils.bootstrap import Container
 
 LOG = logging.getLogger(__name__)
 
 
-@hooks.configure_localstack_container()
-def configure_edge_port(container: Container):
-    ports = [config.EDGE_PORT, config.EDGE_PORT_HTTP]
-    LOG.debug("configuring container with edge ports: %s", ports)
-    for port in ports:
-        if port:
-            container.config.ports.add(port)
-
-
```

Comment:
This is not needed because we use `GATEWAY_LISTEN` in `configure_container`, right? If so, thanks for picking up on this

Reply:
yep, correct!